scipy-kr-2023/category.json

```
@@ -0,0 +1,3 @@
+{
+  "title": "SciPy Korea - 2023"
```

```suggestion
  "title": "SciPy Korea 2023"
```
PyVideo automatically extracts the year from a conference name, and so it does not expect a `-` or other separator character here.